clinica/iotools/converters/adni_to_bids/adni_utils.py

```diff
@@ -1166,7 +1166,12 @@ def create_file(image, modality, total, bids_dir, mod_to_update):
 
         if image.Is_Dicom:
             command = f"dcm2niix -b {generate_json} -z {zip_image} -o {output_path} -f {output_filename} {image_path}"
-            subprocess.run(command, shell=True, stdout=subprocess.DEVNULL)
+            output_dcm2niix = subprocess.run(command, shell=True, capture_output=True)
+            if output_dcm2niix.returncode != 0:
+                cprint(
+                    msg=f'WARNING: errors may have occured.\nOutput:{output_dcm2niix.stdout.decode("utf-8")}\nError:{output_dcm2niix.stderr.decode("utf-8")}',
+                    lvl="warning",
+                )
```

Comment:
We should probably factor this part out since it's used verbatim in 3 different places

Reply:
Also you can drop the `WARNING` prefix in the message. The logger will take care of it.

Reply:
Done !